usr/share/rear/conf/default.conf

```diff
@@ -2887,38 +2887,46 @@ RECREATE_USERS_GROUPS="no"
 ####
 # BACKUP=EXTERNAL
 ##
-# Custom command backup stuff.
+# Custom command backup stetup.
```

Comment:
typo

Reply:
@pcahyna 
thank you for your careful proofreading!

Fixed via
https://github.com/rear/rear/pull/2872/commits/2b1533bc5b21ff0da82b82631b9c6b83408bcb45